front/src/modules/ui/object/field/meta-types/hooks/useCurrencyField.ts

```diff
@@ -40,7 +40,7 @@ export const useCurrencyField = () => {
     ? { amountMicros: 0, currencyCode: '' }
     : !isNaN(Number(fieldInitialValue?.value))
     ? { amountMicros: Number(fieldInitialValue?.value), currencyCode: '' }
```

Comment:
Note: I'm not sure if it's the right place but the FE should probably send a default currency and not empty string? Wdyt? (cc @charlesBochet)

Reply:
yes, let's send USD for now